enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java

```diff
@@ -19,42 +19,49 @@
 package io.crate.beans;
 
 import com.google.common.collect.ImmutableList;
-import io.crate.auth.user.User;
+import com.google.common.collect.ImmutableMap;
 import io.crate.execution.engine.collect.stats.JobsLogs;
-import io.crate.expression.reference.sys.job.JobContext;
-import io.crate.expression.reference.sys.job.JobContextLog;
+import io.crate.metadata.sys.ClassifiedMetrics.Metrics;
 import io.crate.planner.Plan.StatementType;
 import io.crate.planner.operators.StatementClassifier.Classification;
 import org.junit.Test;
 
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-import java.util.UUID;
 
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertThat;
 
 public class QueryStatsTest {
 
+    private static final int HIGHEST_TRACKABLE_VALUE = 600000;
+
     private static final Classification SELECT_CLASSIFICATION = new Classification(StatementType.SELECT);
```

Comment:
Maybe we could add a second classification which has different labels but the same type so that the merge logic is tested?

Reply:
Good point. Pushed fixup